client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -141,7 +143,13 @@ class OrderFulfillment extends Component {
 		}
 	};
 
-	isAddressValidForLabels( address ) {
+	isAddressValidForLabels( address, type ) {
+		if ( isEnabled( 'woocommerce/extension-wcservices/international-labels' ) ) {
+			return (
+				'destination' === type || includes( ACCEPTED_USPS_ORIGIN_COUNTRY_CODES, address.country )
```

Comment:
Would be cool to put this on two lines  - to reduce the long line length (at 100 right now), and to include an explanation for `'destination' === type` since it's not immediately obvious what we're checking.

Reply:
I added a comment in a later PR: https://github.com/Automattic/wp-calypso/pull/25723/files#diff-e6282fa0c7eb967d83aed8923c5e1fcaR151

About the line lenght, there's nothing I can do (I wouldn't have formatted it like this by hand, that's for sure), it's all done automatically by `prettier`.